hazelcast/src/main/java/com/hazelcast/scheduledexecutor/impl/ScheduledTaskStatisticsImpl.java

```diff
@@ -165,8 +165,10 @@ public ScheduledTaskStatisticsImpl snapshot() {
 
     @Override
     public String toString() {
-        return "ScheduledTaskStatisticsImpl{ runs=" + runs + ", createdAt="
-                + createdAt + ", firstRunStart=" + firstRunStart + ", lastRunStart=" + lastRunStart + ", lastRunEnd=" + lastRunEnd
-                + ", lastIdleTime=" + lastIdleTime + ", totalRunTime=" + totalRunTime + ", totalIdleTime=" + totalIdleTime + '}';
+        return "ScheduledTaskStatisticsImpl{ runs=" + runs
```

Comment:
`ScheduledTaskStatisticsImpl{runs=` without a space matches the majority of our `toString()` methods.